_data/students.yml

```diff
@@ -570,8 +570,7 @@
   twitter: guoJLbenjamin
   bio: Cynical
   lat: 1.3553794
-  lng: 103.86774439999999
-
+  lng: 103.86774
```

Comment:
Dont change others info without their consent and  add a - before the name like the other students do please